src/tests/test_viewsGenerationSystem.py

```diff
@@ -119,56 +78,90 @@ def tearDown(self):
 		if Path(DATA_DIR._root.value).exists():
 			shutil.rmtree(DATA_DIR._root.value)
 
-	def _check_expected_addons_added(self):
-		"""
-		Checks that all the data and files in expectedResultsPath match the test output directory.
-		"""
-		testOutputFiles = set(glob.glob(DATA_DIR.OUTPUT + VIEWS_GLOB))
-		expectedFiles = set(glob.glob(DATA_DIR.EXPECTED_RES + VIEWS_GLOB))
-		testOutputFiles = set(f.lstrip(DATA_DIR.OUTPUT) for f in testOutputFiles)
-		expectedFiles = set(f.lstrip(DATA_DIR.EXPECTED_RES) for f in expectedFiles)
-		self.assertEqual(testOutputFiles, expectedFiles)
-		for outputFilename in testOutputFiles:
-			with open(f"{DATA_DIR.OUTPUT}/{outputFilename}", "r") as outputFile:
-				outputFileJson = json.load(outputFile)
-			with open(f"{DATA_DIR.EXPECTED_RES}/{outputFilename}", "r") as expectedFile:
-				expectedResultsJson = json.load(expectedFile)
-			self.assertDictEqual(expectedResultsJson, outputFileJson, msg=outputFilename)
-
-	def _test_transform(self) -> None:
+	def runTransformation(self) -> subprocess.CompletedProcess:
 		"""Generates NVDA API versions so that transform can run.
+		Runs the transformation and raises a CalledProcessError on failure.
 		"""
 		_TestDataGenerator.write_nvdaAPIVersions()
-		subprocess.run(
+		transformProcess = subprocess.run(
 			f"python -m src.transform {DATA_DIR.nvdaAPIVersionsPath} {DATA_DIR.INPUT} {DATA_DIR.OUTPUT}",
-			shell=True
-		).check_returncode()  # Raise CalledProcessError if the exit code is non-zero.
+			shell=True,
+			stderr=subprocess.PIPE
+		)
+		transformProcess.check_returncode()  # Raise CalledProcessError if the exit code is non-zero.
+		return transformProcess
 
 	def test_transform_empty(self):
 		"""Confirms an empty transformation is successful
 		"""
-		self._test_transform()
+		self.runTransformation()
 
 	def test_transform_successfully(self):
-		"""Confirms an transformation of a single addon is successful
+		"""Confirms a transformation of a single addon runs successfully
```

Comment:
From this test it's hard to tell what successfully means. Did you intend it to mean "without error"?

Reply:
```suggestion
		"""Confirms a transformation of a single addon runs and exits without an error exit code
```